armbian.IMGBUILD

```diff
@@ -0,0 +1,60 @@
+COMPRESSZST=(zstd -c -T0 -18 -)
```

Comment:
This seems to just take the Armbian baseimage without any modifications...what am I missing here>

Reply:
the armbian base image is in a weird or incorrect archive format. So i repackage it with this so we can offer a base image that can be easily extracted if users choose to do so.

this is not a critical file, in any case. 

its repackaging the image after it was already downloaded by the other IMGBUILD.

Reply:
removed now